src/main/java/com/aws/greengrass/clientdevices/auth/configuration/CAConfiguration.java

```diff
@@ -132,6 +139,17 @@ private static URI getUri(String rawUri) throws URISyntaxException {
         return uri;
     }
 
+    private static Optional<URI> getCaCertificateChainUriFromConfiguration(Topics certAuthorityTopic)
+            throws URISyntaxException {
+        String certificateChainUri = Coerce.toString(certAuthorityTopic.findOrDefault("", CA_CERTIFICATE_CHAIN_URI));
```

Comment:
using find or default here with an empty string is rather pointless. Coerce to string will give you null if it doesn't exist, which is fine.